src/Gui/DlgSettingsWorkbenchesImp.h

```diff
@@ -31,42 +31,59 @@ class QCheckBox;
 
 namespace Gui {
 namespace Dialog {
-class Ui_DlgSettingsLazyLoaded;
+class Ui_DlgSettingsWorkbenches;
 
 
 /**
- * The DlgSettingsLazyLoadedImp class implements a pseudo-preference page explain why
+ * The DlgSettingsWorkbenchesImp class implements a pseudo-preference page explain why
  * the remaining preference pages aren't loaded yet, and to help the user do so on demand.
  * \author Jürgen Riegel
  */
-class DlgSettingsLazyLoadedImp : public PreferencePage
+class DlgSettingsWorkbenchesImp : public PreferencePage
 {
     Q_OBJECT
 
 public:
-    explicit DlgSettingsLazyLoadedImp( QWidget* parent = nullptr );
-    ~DlgSettingsLazyLoadedImp() override;
+    explicit DlgSettingsWorkbenchesImp( QWidget* parent = nullptr );
+    ~DlgSettingsWorkbenchesImp() override;
 
     void saveSettings() override;
     void loadSettings() override;
 
+    static QStringList getEnabledWorkbenches();
+    static QStringList getDisabledWorkbenches();
```

Comment:
Why are they static functions ?

Reply:
I just copy pasted them from the customize dlg. I think they are static because they are used in Action.cpp.

Reply:
Good reason. :)